### PR TITLE
[MM-39849] Add followers to my runs filter

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This ticket has changed a bit. As discussed in the meeting we decided to just show run followers too(with the participants) when we toggle `my runs only` checkbox. 
Basically `my runs` = `runs I'm following` + `runs I'm participating in`

No UX changes

#### Ticket Link

  Fixes: https://mattermost.atlassian.net/browse/MM-39849

#### Checklist

- ~~[ ] Telemetry updated~~
- ~~[ ] Gated by experimental feature flag~~
- ~~[ ] Unit tests updated~~

### CODE REVIEW

**Comment by stephenvanhemmen** [comment]
Does the Runs page default to having `My runs only` selected? If not, the runs page should only show the user's participating/following runs by default. This is to test the assumption that users use the Runs page to see their specific runs more often than other peoples' runs.

**Comment by iomodo** [comment]
> Does the Runs page default to having `My runs only` selected? If not, the runs page should only show the user's participating/following runs by default. This is to test the assumption that users use the Runs page to see their specific runs more often than other peoples' runs.

It does not. I'll make the change. Thanks @stephenvanhemmen !

**Comment by iomodo** [comment]
`My runs only` will be checked by default in the runs page as well as the usage page

Screen Shot 2021-11-12 at 18 30 51

File: server/sqlstore/playbook_run.go [hunk 1]
```diff
@@ -232,6 +232,24 @@ func (s *playbookRunStore) GetPlaybookRuns(requesterInfo app.RequesterInfo, opti
...
+			FROM IR_Run_Participants as rp
+			WHERE rp.IncidentID = i.ID
+			AND rp.UserID = ?
+			AND rp.IsFollower = ?)`
+		participantFilter := `EXISTS(SELECT 1
+			FROM ChannelMembers AS cm
+			WHERE cm.ChannelId = i.ChannelID
+			AND cm.UserId = ?)`
+		query := fmt.Sprintf("(%s OR %s)", followerFilter, participantFilter)
+		myRunsClause := sq.Expr(query, userIDFilter, true, userIDFilter)
```
Comment (agarciamontoro) [inline server/sqlstore/playbook_run.go:247]: Nit: for the sake of simplicity, can we hardcode the `true` in the followerFilter`'s `rp.IsFollower` condition? That way we can remove one parameter to `myRunsClause`.

**Review by agarciamontoro — APPROVED** [review]
Looks good :tada: Just a minor, non-blocking, nit. Thanks!

**Comment by stephenvanhemmen** [comment]
> `My runs only` will be checked by default in the runs page as well as the usage page
> 
> Screen Shot 2021-11-12 at 18 30 51

Just one more thing, could we keep this option unchecked by default on the Usage page but checked by default on the Runs list? Users seem to have different uses for these two lists, even though they both display runs. Sorry about that, I should have specified. Thanks!

**Comment by iomodo** [comment]
> Just one more thing, could we keep this option unchecked by default on the Usage page but checked by default on the Runs list? Users seem to have different uses for these two lists, even though they both display runs. Sorry about that, I should have specified. Thanks!

No worries, done

**Review by stephenvanhemmen — APPROVED** [review]

**Comment by iomodo** [comment]
Adjusted input field size
Screen Shot 2021-11-17 at 18 31 16

**Review by crspeller — APPROVED** [review]

**Pull request merged**